manifests/init.pp

```diff
@@ -1,244 +1,241 @@
-  Stdlib::Httpsurl                    $repo_apt_gpgkey                    = $rundeck::params::repo_apt_gpgkey,
-  String                              $repo_apt_keyserver                 = $rundeck::params::repo_apt_keyserver,
-  Boolean                             $rss_enabled                        = $rundeck::params::rss_enabled,
-  Hash                                $security_config                    = $rundeck::params::security_config,
-  String                              $security_role                      = $rundeck::params::security_role,
-  Optional[String]                    $server_web_context                 = undef,
-  Optional[String]                    $service_config                     = undef,
-  Stdlib::Absolutepath                $service_logs_dir                   = $rundeck::params::service_logs_dir,
-  String                              $service_name                       = $rundeck::params::service_name,
-  Boolean                             $service_restart                    = true,
-  Optional[String]                    $service_script                     = undef,
-  Enum['stopped', 'running']          $service_ensure                     = $rundeck::params::service_ensure,
-  Integer                             $session_timeout                    = $rundeck::params::session_timeout,
-  Boolean                             $ssl_enabled                        = $rundeck::params::ssl_enabled,
-  Stdlib::Port                        $ssl_port                           = $rundeck::params::ssl_port,
-  Stdlib::Absolutepath                $truststore                         = $rundeck::params::truststore,
-  String                              $truststore_password                = $rundeck::params::truststore_password,
-  String                              $user                               = $rundeck::params::user,
-  String                              $group                              = $rundeck::params::group,
-  Boolean                             $manage_user                        = $rundeck::params::manage_user,
-  Boolean                             $manage_group                       = $rundeck::params::manage_group,
-  Optional[Integer]                   $user_id                            = undef,
-  Optional[Integer]                   $group_id                           = undef,
-  String                              $file_default_mode                  = $rundeck::params::file_default_mode,
-  Boolean                             $security_roles_array_enabled       = $rundeck::params::security_roles_array_enabled,
-  Array                               $security_roles_array               = $rundeck::params::security_roles_array,
-  Hash[String,String]                 $storage_encrypt_config             = {},
-) inherits rundeck::params {
-  validate_rd_policy($acl_policies)
+  Stdlib::Absolutepath           $override_dir,
```

Comment:
I would also eliminate the alignment spaces between the data type and variable name, just like you did between the variable and the default value.

We had a discussion about this at https://github.com/voxpupuli/voxpupuli.github.io/pull/322.